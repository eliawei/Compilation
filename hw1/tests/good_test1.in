and And
or Or
not Not
true True
false False
return Return
if If
if IF
else Else
else ELSE
while While
break Break
break BREAK
continue Continue
continue CONTINUE
1 / 2;