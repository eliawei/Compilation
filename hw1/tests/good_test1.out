1 AND and
1 ID And
2 OR or
2 ID Or
3 NOT not
3 ID Not
4 TRUE true
4 ID True
5 FALSE false
5 ID False
6 RETURN return
6 ID Return
7 IF if
7 ID If
8 IF if
8 ID IF
9 ELSE else
9 ID Else
10 ELSE else
10 ID ELSE
11 WHILE while
11 ID While
12 BREAK break
12 ID Break
13 BREAK break
13 ID BREAK
14 CONTINUE continue
14 ID Continue
15 CONTINUE continue
15 ID CONTINUE
16 NUM 1
16 BINOP /
16 NUM 2
16 SC ;
